test/server_selection_logging/operation-id.json

```diff
@@ -197,7 +197,7 @@
               }
             },
             {
-              "level": "debug",
+              "level": "info",
```

Comment:
How are we passing this test? We actually (intentionally) log this at info level. See PYTHON-4261.

Reply:
We're logging that line at "info" level while the spec test claims it should be at "debug" level. I would expect the test to fail. This indicates theres a bug in the test runner that we need to fix.